pkg/config/config.go

```diff
@@ -270,6 +272,25 @@ func NewConfig(confString string, c *cli.Context) (*Config, error) {
 			conf.RTC.ICEPortRangeEnd = 60000
 		}
 	}
+	
+	// set defaults for Turn relay if none are set
+	if conf.TURN.RelayPortRangeStart == 0 {
+		// to make it easier to run in dev mode/docker, default to two ports
+		if conf.Development {
```

Comment:
this bit of code is a bit confusing to read. could we make it so that:

when conf.Development:
* 30000-30002
otherwise
* 30000-40000

I know you are keeping the original behavior of 1024+. I think 10k ports is more than enough for TURN on a single instance, so this seems like a more reasonable number to use.

Reply:
I changed the way I'm checking for missing configuration values and applying defaults. Please check if I understood correctly your point